src/main/java/icons/MacheteIcons.java

```diff
@@ -0,0 +1,9 @@
+package icons;
+
+import com.intellij.openapi.util.IconLoader;
+import javax.swing.Icon;
+
+@SuppressWarnings("regexp")
```

Comment:
:hushed: what's RegexpChecker screaming here?

Reply:
It's screaming about interface name (should starts with `I`) but as you suggested I just transformed it to final class